src/utils/authentication.js

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+
+/**
+ *
+ * @param e Element
+ * @param location useLocation()
+ * @param navigate useNavigation()
+ */
+const loginHandler = (e, location, navigate) => {
+	e.preventDefault();
+
+	const formData = new FormData(e.target);
```

Comment:
Why not make use of states. Why FormData? Any particular reason?

Reply:
Yeah I was testing the function with formData initialy and then I created the states and didn't change this function, Thanks will make that change